doc/external-libs/trilinos.html

```diff
@@ -87,6 +88,7 @@ <h5>Installing Trilinos</h5>
     -DTrilinos_ENABLE_MueLu=ON                       \
     -DTrilinos_ENABLE_ML=ON                          \
     -DTrilinos_ENABLE_ROL=ON                         \
+    -DTrilinos_ENABLE_Zoltan=ON                      \
```

Comment:
is this really needed? We do not specify it in https://github.com/dealii/candi/blob/master/deal.II-toolchain/packages/trilinos.package#L146 for example but it certainly builds zoltan.

Reply:
Are you saying that with the existing flags, the Trilinos dependency tracking determines that it needs to enable Zoltan anyway? Either way, I think we all just copy-paste these flags, so why not err on the safe side and ask users to enable it explicitly.

Reply:
i would keep it, it can't harm

Reply:
> Are you saying that with the existing flags, the Trilinos dependency tracking determines that it needs to enable Zoltan anyway?

Yes.